client/my-sites/plan-features/actions.jsx

```diff
@@ -22,23 +23,26 @@ const PlanFeaturesActions = ( {
 	translate
 } ) => {
 	let upgradeButton;
-	const className = classNames( {
-		'plan-features__actions-button': true,
-		'is-current': current,
-		'is-primary': popular && ! isPlaceholder,
-	} );
+	const classes = classNames(
+		'plan-features__actions-button',
```

Comment:
You know what, I have actually suggested this to you 13 days ago in your PR, but you didn't listen :D. Proof:

![selection_019](https://cloud.githubusercontent.com/assets/4988512/17217611/3abfa552-54e5-11e6-848f-d18980e40e92.jpg)